lxc/list.go

```diff
@@ -570,6 +579,45 @@ func (c *cmdList) parseColumns(clustered bool) ([]column, bool, error) {
 	return columns, needsData, nil
 }
 
+func (c *cmdList) baseImageAliasColumnData(cInfo api.ContainerFull) string {
+	var err error
+
+	if localServer == nil {
+		localServer, err = c.global.conf.GetImageServer("local")
+		if err != nil {
+			return ""
+		}
+	}
```

Comment:
The image should be searched for on the same server that `lxc list` itself is running against, it makes no sense to have `lxc list blah:` look for the images on a likely empty local server.